src/modules/history/components/HistoryHeader.tsx

```diff
@@ -64,6 +66,30 @@ export const HistoryHeader: React.FC = () => {
 							</Button>
 						</>
 					}
+					middle={
+						<Typography variant="overline">
+							{serviceId !== null ? (
+								service ? (
+									<>
+										{I18N.translate('history')} {I18N.translate('for')}{' '}
```

Comment:
I can't see any reason for not combining these two i18n-strings into one. Besides prepositions are often hard to translate because they are dependent on the context. (In this case the Norwegian translation should be "for", identical as the English).

Reply:
I thought it would be repetitive to have "history" and "historyFor", but you're right, it's best to use full sentences. Just pushed a commit that merges them.